### ISSUE TITLE
[Suggestion] "Broken-up stream" function

### ISSUE THREAD

**Issue by Samario1** [LookItsBlu/Soundodger-Tools#7]
i.e. you pick a start time, end time, and sparseness (say, 1 is every frame (16 2/3ms), 2 is every other frame etc), and it generates the sdon/xml required to have a "stream" of bullets that lasts that long

However it's implemented would work

**Comment by Samario1** [comment]
yeah, for things like having a denser/sparser stream or varying the bullet type in the middle of the stream

**Comment by LookItsBlu** [comment]
Will consider for the official release of Soundodger Tools, I'm all in favor of this idea
